### ISSUE TITLE
Help / tutorial

### ISSUE THREAD

**Issue by ceholden** [ceholden/TSTools#52]
Would be nice to have a "Getting started" dialog similar to what is shown in some Android applications that explains what all the buttons in TSTools are used for. We could write some descriptions of each panel/tab/etc. and explain functionality. On initial launch of TSTools, or when the user asks for it via the options menu (#51), show popups located above each panel like, "This is the images table that contains all images in a timeseries. It lists images by ID and date and can contain various metadata about each image. Click on a row to add the image". Have a "next" button that goes to the next one. 

Depends on #22 to allow users to permanently disable tutorial.

**Comment by acocac** [comment]
Is any tutorial available for this tool? Thank you
